port.js

```diff
@@ -169,52 +169,45 @@ Port.prototype.stop = function stop() {
 };
 
 Port.prototype.request = function request() {
-    var port = this;
-    var $meta = (arguments.length && arguments[arguments.length - 1]) || {};
+    var $meta = arguments.length && arguments[arguments.length - 1];
+    if (!arguments.length) {
+        return Promise.reject(errors.missingParams());
+    } else if (!$meta) {
```

Comment:
This actually is not very correct in my oppinion, because if there's just 1 argument - it will be treated as $meta. (The same problem refers to the master branch of course.) Maybe we should add an additional check that arguments.length is > 1